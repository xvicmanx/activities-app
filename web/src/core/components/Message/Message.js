import React from 'react';
import './Message.css';

const Message = ({ children }) => {
  return <div className="Message">{children}</div>;
};

export default Message;
