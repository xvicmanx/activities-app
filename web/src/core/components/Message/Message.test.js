import React from 'react';
import renderer from 'react-test-renderer';
import Message from './Message';

describe('Message', () => {
  it('renders properly', () => {
    const component = renderer.create(
      <Message>Hello my friend!</Message>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});