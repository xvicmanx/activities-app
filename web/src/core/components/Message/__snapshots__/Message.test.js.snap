// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders properly 1`] = `
<div
  className="Message"
>
  Hello my friend!
</div>
`;
